packages/storybook-block/src/types.ts

```diff
@@ -58,11 +58,11 @@ export const borderStyles: Record<StorybookBorderStyle, string> = {
     [StorybookBorderStyle.Dashed]: 'dashed',
 };
 
-export const borderRadiusClasses: Record<StorybookBorderRadius, string> = {
-    [StorybookBorderRadius.None]: 'tw-rounded-none',
-    [StorybookBorderRadius.Small]: 'tw-rounded',
-    [StorybookBorderRadius.Medium]: 'tw-rounded-md',
-    [StorybookBorderRadius.Large]: 'tw-rounded-lg',
+export const borderRadiuses: Record<StorybookBorderRadius, string> = {
+    [StorybookBorderRadius.None]: '0px',
+    [StorybookBorderRadius.Small]: '0.25rem',
+    [StorybookBorderRadius.Medium]: '0.375rem',
+    [StorybookBorderRadius.Large]: '0.5rem',
```

Comment:
Where do these values come from? We have a `radiusStyleMap` with different values [in the shared package](https://github.com/Frontify/guideline-blocks/blob/5541381e3f11617502462f14b7e584a21c01b813/packages/shared/src/settings/types.ts#L24) (some other blocks are using the values from the shared package). Should we maybe adjust these values there too?

Reply:
I just took the values that were behind the tailwind classes.
But the radiusStyleMap makes more sense. I changed it and now the radiusStyleMap is used. 😊